utilities/PerformanceTesting/wait-for-cluster.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+name=$1
+
+echo "Waiting for infra cluster to be ready"
+
+while true; do
+    ready="$({ infractl get "$name" | awk '{if ($1 == "Status:" && $2 == "READY") print}' || true; } | wc -l)"
+    if ((ready == 1)); then
+        break
+    fi
+done
```

Comment:
Is there no need for a `sleep` between retries? Nor do we need a timeout in case this takes too long?

Reply:
Add a sleep and set the maximum number of retries to 10,000. This sometimes takes more than 40 minutes.